library/src/morpheus-master/sass/modules/grader/_base.scss

```diff
@@ -233,12 +233,30 @@ sakai-grader {
       }
     }
 
-    sakai-file-picker {
-      margin-left: 10px;
+    sakai-grader-file-picker {
 
       .sakai-file-picker-list {
         margin-top: 5px;
         padding-left: 10px;
```

Comment:
```suggestion
        margin-top: calc(#{$standard-spacing} / 4);
        padding-left: calc(#{$standard-spacing} / 2);
```

Reply:
Would you prefer to expose standard spacing as a CSS Custom Property and access it directly in the component vs style it in scss?